src/NuGet.Clients/PackageManagement.UI/Models/SearchResult.cs

```diff
@@ -47,9 +50,17 @@ internal class SearchResult<T> : IEnumerable<T>
     /// </summary>
     internal static class SearchResult
     {
-        public static SearchResult<T> FromItems<T>(params T[] items) => new SearchResult<T> { Items = items };
+        public static SearchResult<T> FromItems<T>(params T[] items) => new SearchResult<T>
+        {
+            Items = items,
+            RawItemsCount = items.Length
```

Comment:
nit: if you keep a , at the end of this line. Future additions keep the diff smaller